usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateAllRequiredFilePresence.java

```diff
@@ -35,12 +37,15 @@ public ValidateAllRequiredFilePresence(final GtfsSpecRepository specRepo,
         this.resultRepo = resultRepo;
     }
 
-    public void execute() {
+    public List<String> execute() {
         if (!rawFileRepo.getFilenameAll().containsAll(specRepo.getRequiredFilenameList())) {
 
             specRepo.getRequiredFilenameList().stream()
                     .filter(requiredFile -> !rawFileRepo.getFilenameAll().contains(requiredFile))
                     .forEach(missingFile -> resultRepo.addNotice(new MissingRequiredFileNotice(missingFile)));
+
+            return specRepo.getRequiredFilenameList();
```

Comment:
I don't think the same line of code should be within and outside of the IF statement here?

Reply:
I agree, will modify.